tests/test_intervals.py

```diff
@@ -101,46 +101,78 @@ def test_clopper_pearson_interval(hist_fixture):
 
 
 def test_ratio_uncertainty(hist_fixture):
-    hist_1, hist_2 = hist_fixture
+    num, denom = np.meshgrid(np.array([0, 1, 4, 512]), np.array([0, 1, 4, 512]))
```

Comment:
As you're creating this meshgrid then I don't think you use the `hist_fixture` anywhere else and it should be removed from the test call in the line above.